OoyalaSkinSampleApp/app/src/main/java/com/ooyala/sample/players/MultiAudioActivity.java

```diff
@@ -0,0 +1,90 @@
+package com.ooyala.sample.players;
+
+import android.os.Bundle;
+import android.util.Log;
+
+import com.ooyala.android.OoyalaNotification;
+import com.ooyala.android.OoyalaPlayer;
+import com.ooyala.android.PlayerDomain;
+import com.ooyala.android.configuration.Options;
+import com.ooyala.android.player.exoplayer.multiaudio.AudioTrack;
+import com.ooyala.android.skin.OoyalaSkinLayoutController;
+import com.ooyala.sample.R;
+
+import java.util.Observable;
+import java.util.Set;
+
+/**
+ * This activity illustrates how you can use multi audio methods
+ *
+ */
+public class MultiAudioActivity extends AbstractHookActivity {
+
+  private AudioTrack currentAudioTrack;
+  private Set<AudioTrack> audioTracks;
+
+	@Override
+	public void onCreate(Bundle savedInstanceState) {
+		super.onCreate(savedInstanceState);
+		setContentView(R.layout.player_skin_simple_layout);
+		completePlayerSetup(asked);
+	}
+
+	@Override
+	void completePlayerSetup(boolean asked) {
+		if (asked) {
+			// Get the SkinLayout from our layout xml
+			skinLayout = findViewById(R.id.ooyalaSkin);
+
+			// Create the OoyalaPlayer, with some built-in UI disabled
+			PlayerDomain playerDomain = new PlayerDomain(domain);
+			Options options = new Options.Builder()
+				.setShowNativeLearnMoreButton(false)
+				.setShowPromoImage(false)
+				.setUseExoPlayer(true)
+				.build();
+
+			player = new OoyalaPlayer(pcode, playerDomain, options);
+			playerLayoutController = new OoyalaSkinLayoutController(getApplication(), skinLayout, player);
+
+			//Add observer to listen to fullscreen open and close events
+			playerLayoutController.addObserver(this);
+			player.addObserver(this);
+
+			if (!player.setEmbedCode(embedCode)) {
+				Log.e(TAG, "Asset Failure");
+			}
+
+			player.setDefaultAudioLanguage("eng");
+			player.setAudioTrack("Track name");
+		}
+	}
+
+  @Override
+  public void update(Observable arg0, Object argN) {
+    super.update(arg0, argN);
+	  final String arg1 = OoyalaNotification.getNameOrUnknown(argN);
+
+    // MULTI_AUDIO_ENABLED_NOTIFICATION_NAME is called once on a video start
+    if (arg1 == OoyalaPlayer.MULTI_AUDIO_ENABLED_NOTIFICATION_NAME) {
+      audioTracks = player.getAvailableAudioTracks();
+      if (audioTracks != null) {
+        for (AudioTrack track : audioTracks) {
+          if (track.getLanguage().equals("eng")) {
+//            player.setAudioTrack(track);
+            break;
+          }
+        }
+      }
+      AudioTrack defaultAudioTrack = player.getDefaultAudioTrack();
```

Comment:
Can you comment these lines saying that this is how you would get the default track and language